javascript/2018_CALL_FOR_TALKS.md

```diff
@@ -0,0 +1,41 @@
+
+If you're reading through the topics and are (even the *tiniest* bit) interested in giving a brief talk on one of them (or a relevant subject), contact Sarah Joy ([twitter](http://twitter.com/superissarah) or [email](mailto:sarahjoy@alchemycodelab.com)) or Haley ([twitter](https://twitter.com/haley_elder)) ASAP.
+
+## Monthly Topics
+Our plan is to spend the first half of the year focusing on core JavaScript concepts. The second half, we'll focus on using JavaScript related tools like build tools and frameworks. However, these topics aren't set in stone and if you'd like to present on something similar or related, please ask.
+
+| Meetup Date | Topic | Looking for presenters |
+| --- | --- | --- |
+| January 31 | Sharing, finding, and building resources |
+| February 28 | JavaScript and your browser: browser API's | 👀
+| March 14 | Hackathon Prep! | 
+| April 11 | Scope, and Closures, and Hoisting, oh my | 👀
+| May 9 | Promises, promises, promises | 👀
+| June 13 | ES6 and probably also ES8 | 👀
+| July 11 | Using build tools to up your CSS game | 
+| August 8 | Typescript | 👀
+| September 12 | Intro to App Architecture Patterns and JS Frameworks | 👀
+| October  10| React  | 👀
+| November 14 | Vue | 👀
+
+## What we're looking for presentation-wise
+
+- Someone to enlighten us about all the methods of the brower's console object.
+- Someone to enthusiastically tell us how they broke (and fixed) their promises.
+- Someone to share their analogies for MVC and MVVM and other design patterns.
+- Someone to share their growing collection of tips and tricks for converting ES5 to ES6.
+- Someone to live draw the call stack as their script unfolds.
+- Someone to tell us JavaScript puns for 8 minutes.
+- Someone to sing to us why Typescript is a lifesaver.
+
+The above examples are very specific, but also very made up. We're looking for women interested in either teaching or sharing their experience learning about a specific JavaScript topic, or as often times happens, both!
+
+Presentations should be short (10 minutes or less), and we will happily work with you to refine your talk.
+
+## Okay, real talk
+I know you had a moment of inspiration recently—a slight lift in your heart/chest area—little light bulb emojis in your eyes—if you're thinking this isn't your time to try, I'm telling you it is—come share your light with us.
+
+Again, if you are (even the *tiniest* bit) interested in giving a brief talk on one of the above topics (or a relevant subject), contact Sarah Joy ([twitter](http://twitter.com/superissarah) or [email](mailto:sarahjoy@alchemycodelab.com)) or Haley ([twitter](https://twitter.com/haley_elder)) ASAP.
```

Comment:
@haleyelder Can I add an email address for you here?

Reply:
sure! you can use hello@haleyelder.com